{"url":"https://api.github.com/repos/elastic/elasticsearch/issues/27027","repository_url":"https://api.github.com/repos/elastic/elasticsearch","labels_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27027/labels{/name}","comments_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27027/comments","events_url":"https://api.github.com/repos/elastic/elasticsearch/issues/27027/events","html_url":"https://github.com/elastic/elasticsearch/issues/27027","id":265959273,"node_id":"MDU6SXNzdWUyNjU5NTkyNzM=","number":27027,"title":"Typo in exception error message","user":{"login":"cfx1","id":9679270,"node_id":"MDQ6VXNlcjk2NzkyNzA=","avatar_url":"https://avatars2.githubusercontent.com/u/9679270?v=4","gravatar_id":"","url":"https://api.github.com/users/cfx1","html_url":"https://github.com/cfx1","followers_url":"https://api.github.com/users/cfx1/followers","following_url":"https://api.github.com/users/cfx1/following{/other_user}","gists_url":"https://api.github.com/users/cfx1/gists{/gist_id}","starred_url":"https://api.github.com/users/cfx1/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/cfx1/subscriptions","organizations_url":"https://api.github.com/users/cfx1/orgs","repos_url":"https://api.github.com/users/cfx1/repos","events_url":"https://api.github.com/users/cfx1/events{/privacy}","received_events_url":"https://api.github.com/users/cfx1/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2017-10-17T00:51:55Z","updated_at":"2017-10-18T13:54:17Z","closed_at":"2017-10-18T13:54:17Z","author_association":"NONE","active_lock_reason":null,"body":"<!-- Bug report -->\r\n\r\nElasticsearch 5.6.3\r\n\r\nOpenJDK Runtime Environment (build 1.8.0_144-b01)\r\nOpenJDK Server VM (build 25.144-b01, mixed mode)\r\n\r\nLinux localhost.localdomain 2.6.32-696.13.2.el6.i686 SMP Thu Oct 5 20:42:25 UTC 2017 i686 i686 i386 GNU/Linux\r\n\r\nIndicesAliasesRequest::ConstructingObjectParser has a typo in the following exception message:\r\n`throw new IllegalArgumentException(\"Too many operations declared in on opeation entry\");`\r\n","closed_by":{"login":"nik9000","id":215970,"node_id":"MDQ6VXNlcjIxNTk3MA==","avatar_url":"https://avatars2.githubusercontent.com/u/215970?v=4","gravatar_id":"","url":"https://api.github.com/users/nik9000","html_url":"https://github.com/nik9000","followers_url":"https://api.github.com/users/nik9000/followers","following_url":"https://api.github.com/users/nik9000/following{/other_user}","gists_url":"https://api.github.com/users/nik9000/gists{/gist_id}","starred_url":"https://api.github.com/users/nik9000/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nik9000/subscriptions","organizations_url":"https://api.github.com/users/nik9000/orgs","repos_url":"https://api.github.com/users/nik9000/repos","events_url":"https://api.github.com/users/nik9000/events{/privacy}","received_events_url":"https://api.github.com/users/nik9000/received_events","type":"User","site_admin":false},"performed_via_github_app":null}